src/app/forms/input/input.html

```diff
@@ -0,0 +1,356 @@
+                <input type="checkbox" name="checkbox1" id="{{layout}}-checkbox1" placeholder="Enter value here" class="clr-checkbox"> 
+                <label for="{{layout}}-checkbox1" class="clr-control-label">
+                    option 1
+                </label>
+                <input type="checkbox" name="checkbox2" id="{{layout}}-checkbox2" placeholder="Enter value here" class="clr-checkbox"> 
+                <label for="{{layout}}-checkbox2" class="clr-control-label">
+                    option 2
+                </label>
+                <input type="checkbox" name="checkbox3" id="{{layout}}-checkbox3" placeholder="Enter value here" class="clr-checkbox"> 
+                <label for="{{layout}}-checkbox3" class="clr-control-label">
+                    option 3
+                </label>
+            </div>
+            <!-- IMPORTANT DIFFERENCE IN STRUCTURE! ICON IS NOT PART OF THE INPUT WRAPPER -->
+            <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+            <span class="clr-subtext">Helper Text</span>
+        </div>
+    </div>
+
+    <div class="clr-form-control" [ngClass]="{row: grid}">
+        <label class="clr-control-label"[ngClass]="{'col-xs-12': grid, 'col-lg-2': grid && layout !== 'vertical'}">
+            Basic checbox, error</label>
+            <div class="clr-control-container clr-error" [ngClass]="{'col-xs-12': grid, 'col-lg-10': grid && layout !== 'vertical'}">
+                <div class="clr-checkbox-wrapper">
+                    <input type="checkbox" name="checkbox4" id="{{layout}}-checkbox4" placeholder="Enter value here" class="clr-checkbox"> 
+                    <label for="{{layout}}-checkbox4" class="clr-control-label">
+                        option 1
+                    </label>
+                    <input type="checkbox" name="checkbox5" id="{{layout}}-checkbox5" placeholder="Enter value here" class="clr-checkbox"> 
+                    <label for="{{layout}}-checkbox5" class="clr-control-label">
+                        option 2
+                    </label>
+                    <input type="checkbox" name="checkbox6" id="{{layout}}-checkbox6" placeholder="Enter value here" class="clr-checkbox"> 
+                    <label for="{{layout}}-checkbox6" class="clr-control-label">
+                        option 3
+                </label>
+            </div>
+            <!-- IMPORTANT DIFFERENCE IN STRUCTURE! ICON IS NOT PART OF THE INPUT WRAPPER -->
+            <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
+            <span class="clr-subtext">Helper Text</span>
+        </div>
+    </div>
+    
+    <div class="clr-form-control" [ngClass]="{row: grid}">
+        <label class="clr-control-label" [ngClass]="{'col-xs-12': grid, 'col-lg-2': grid && layout !== 'vertical'}">
+            Inline checkbox</label>
+        <div class="clr-control-container" [ngClass]="{'col-xs-12': grid, 'col-lg-10': grid && layout !== 'vertical'}">
```

Comment:
Missing `.clr-error` class. Noting this because you have the error icon at the end.

Reply:
I've put the error icon everywhere, even if its not active. The ones with the `.clr-error` class are the only ones that should show the error icon, so I needed to verify the icon was hidden by default.